### ISSUE TITLE
Is there a way to customize the login attribute mapping (required for the Admin installation)

### ISSUE THREAD

**Issue by pgoldweic** [MicrosoftDocs/microsoft-365-docs#5865]
The login attribute selection is limited, and it does not work for institutions where the Azure Active Directory identities do not correspond to any of the provided options. Is there a way to customize this identity mapping? If not, is this feature in the Microsoft roadmap?

**Comment by yogkumgit** [comment]
@pgoldweic Please indicate which document you are referring to and what is your feedback . Thanks

**Comment by msbemba** [comment]
@pgoldweic Are you referring to https://docs.microsoft.com/en-us/azure/active-directory/hybrid/how-to-connect-install-custom ?

Thanks
Sri

**Comment by pgoldweic** [comment]
Hi,
Thanks for replying. I was actually referring to the Canvas-specific install documents, wrote just a week or two before I posted (and the state of the software at that time) – sorry I did not mention Canvas in the comment-. However, since I wrote the comment, Microsoft released an update to the software that is now accepting some ‘custom’ mappings (by adding a suffix to the login attribute), which happens to be enough for our installation requirements.
Thanks!

-Patricia
